nflow-engine/src/main/java/io/nflow/engine/workflow/definition/WorkflowStateType.java

```diff
@@ -26,6 +26,12 @@
    */
   normal(false, WorkflowInstanceStatus.inProgress),
 
+  /**
+   * State for waiting something outside this instance to happen, including (but not limited to) child workflow instances to go to
+   * end state. When a child workflow finishes and parent is in wait state, parent is automatically woken up.
+   */
+  wait(false, WorkflowInstanceStatus.inProgress),
```

Comment:
Does nflow-explorer need changes to support this? e.g. own color for 'waiting' status?

Reply:
Own color added